examples/pubsub-ts/index.ts

```diff
@@ -18,12 +18,14 @@ const randomString = new random.RandomString("name", {
 const topic = new google.pubsub.v1.Topic("topic", {
     project,
     topicId: randomString.result,
+    name: randomString.result,
```

Comment:
Did this work before?

Reply:
Surprisingly it did, even though the property is marked as required. There may be other instances of that but I'd rather follow what the docs says even if it's overly strict.

Reply:
I wonder if it would be worth filing an upstream issue for that discrepancy?